cli/cmdlineparser.cpp

```diff
@@ -397,6 +397,9 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
                 }
             }
 
+            else if (std::strncmp(argv[i], "--fsigned-char", 14) == 0)
```

Comment:
I suggest `(std::strcmp(argv[i], "--fsigned-char") == 0)`

Reply:
Good catch. That was quite a stupid copy and paste error. Luckily not something with security implications.